catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/monitor/impl/WorkspaceServiceImpl.java

```diff
@@ -184,24 +230,9 @@ private QueryRequest createQueryRequest(List<Filter> filters) {
    * @return query request
    */
   private QueryRequest createQueryRequest(Filter filter) {
-    return new QueryRequestImpl(new QueryImpl(filter), createProperties());
-  }
-
-  /**
-   * Create the filters for getting workspace metacards based on workspace ids.
-   *
-   * @param workspaceIds set of workspace ids
-   * @return list of filters
-   */
-  private List<Filter> createFilters(Set<String> workspaceIds) {
-    return workspaceIds
-        .stream()
-        .map(
-            id ->
-                filterBuilder.allOf(
-                    filterService.buildMetacardIdFilter(id),
-                    filterService.buildWorkspaceTagFilter()))
-        .collect(Collectors.toList());
+    QueryImpl query = new QueryImpl(filter);
```

Comment:
also considering setting a SortBy (primary and secondary sort). if there are more workspaces than maxSubscriptions. The result would be inconsistent and differ for every query.

Reply:
done. copied same approach as #3341